rai/common.hpp

```diff
@@ -110,13 +110,14 @@ std::unique_ptr<rai::block> deserialize_block (MDB_val const &);
 /**
  * Latest information about an account
  */
+#pragma pack(push, 1)
```

Comment:
It may not be a problem in this case, but this does mean we need to worry about unaligned access as we maintain these classes (i.e. no atomics in the future in these structs, no assignment from ptr-to-member, and for ARM, slow code for unaligned access may be generated). 

An alternative is maybe to pack manually instead of std::copy'ing the struct pointer.